### PR TITLE
[3258] Improve crash reports for Sequencer

### PR DESCRIPTION
**Pull request by humphd**

### CODE REVIEW

File: src/crashreporter.js [hunk 1]
```diff
@@ -169,6 +169,7 @@ define( [ "dialog/dialog", "util/xhr", "util/uri" ], function( Dialog, XHR, URI
               butterVersion: butterVersion,
               // Grab the last 5 null dom nodes, if any
               nullDomNodes: _nullDomList.slice( -5 ).join( ", " ) || "N/A",
+              projectData: JSON.stringify( butter.project.data ),
```
Comment (jbuck) [inline src/crashreporter.js]: could you add the pretty-printing option? `JSON.stringify( data, null, 2);`

**Comment by dseif** [comment]
😢

**Comment by humphd** [comment]
@dseif hey, nice to see you!  I wondered what happened to you.  Figures I have to go cleaning up my old PRs to find you.

**Comment by dseif** [comment]
@humphd haha yea been too long. I'm still working on video stuff and still using popcorn. So not too much has changed.

Do you still hang out in the Seneca IRC channels?

**Comment by humphd** [comment]
Still working on Popcorn stuff?  Crazy, I had no idea! cc @brettgaylor.

I'm on on Twitter and GitHub @humphd.  No more irc for me.

**Comment by dseif** [comment]
no irc?!? Slack?? Crazy. I stopped using twitter years ago. I'll shoot you an email instead of having a convo in a 5 year old PR haha.

**Comment by brettgaylor** [comment]
TFW the unpopped kernel at the bottom of the popcorn maker finally pops 5 YEARS LATER.

**Pull request closed without merging**